app/assets/javascript/src/shared/bootstrap_activators.js

```diff
@@ -16,3 +16,37 @@ $(document).on("click", ".toggle-chevron-content", function toggleIcons() {
   "turbo:submit-start",
   "turbo:frame-render",
 ].forEach((eventName) => document.addEventListener(eventName, () => removeTooltips()));
+
+$(document).on(
```

Comment:
https://github.com/freeats/freeats/pull/135#discussion_r1838239007:
> Do we rally need to copy this js code? Won't it work out of the box since we load bootstrap? (actually, since we migrate to tabler, it should be tabler's style)

I think it is needed:
- [The section of bootstrap docs](https://getbootstrap.com/docs/5.3/forms/validation/#custom-styles) has such an example with JS:
- [There is a code for this in the hub](https://github.com/toughbyte/hub/blob/3bd6460ea7c409ecfba724859f8a2497689da680/app/assets/javascript/src/shared/bootstrap_activators.js#L20-L29)
-

Reply:
I'm not really in the context of the issue, but maybe use server-side validation with bootstrap and don't add js code for it?